src/style.css

```
@@ -1,3 +1,62 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --link-active-color: #00acdc;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+}
```

```suggestion
.header {
  display: flex;
  background-color: #fff;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
}
```

And you can remove margins from logo and nav